contracts/bprotocol/avatar/Avatar.sol

```diff
@@ -18,8 +22,9 @@ contract Avatar is AbsComptroller, AbsCToken {
      * @dev Initialize the contract variables
      * @param _registry Registry contract address
      */
-    function initialize(address _registry) external {
+    function initialize(address _registry, address comp, address compVoter) external {
         _initAvatarBase(_registry);
+        CompLike(comp).delegate(compVoter);
```

Comment:
We can read the `comp` and `compVoter` both from the registry contract. So that we don't need to pass these as arguments.

Reply:
yes, but it saves gas (external call) to pass it like this.